cachix/src/Cachix/Client/Push.hs

```diff
@@ -46,13 +52,17 @@ import Servant.Auth ()
 import Servant.Auth.Client
 import Servant.Client.Streaming
 import Servant.Conduit ()
+import System.Environment (lookupEnv)
 import qualified System.Nix.Base32
 
+data PushSecretKey
+  = PushToken Token
+  | PushSigningKey SigningKey
```

Comment:
Most tokens are usually generated by cryptographic algorithm (even in this project), so that definition doesn't hold water in practice.

Either way, I'm not interested into bike shedding, happy to rename it.

Reply:
I wasn't trying to bike shed, although my tone was perhaps a bit pedantic. Even if words have wiggle room, it's good to avoid that.